index.html

```diff
@@ -216,40 +141,158 @@ <h2>Multi-Post Stories</h2>
+  
+  <!--PLACE HOLDER IMAGE2-->
+  <div class="place-holder-left">
+  <img src="./img/ImgPlaceholder.png" alt="place holder">
+  </div>
+  <!-- MUTLI POST STORIES -->
+  <div class="title-post-left">
+  <h2>Multi-Post Stories</h2>
+  </div>
+  <div class="paragraph-left">
+  <p>A daily selection of privately 
+    personalized reads;<br> no accounts 
+    or sign-ups required. has been the <br> 
+    industry's standard dummy text 
+    ever since the <br> 1500s, when an
+    unknown printer took a standard <br>
+    dummy text.</p>
+  </div>
+
+  <!-- languajes de programacion -->
+  <ul class="programming-languages-left">
+  <li><a href="">CSS</a></li>
+  <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+  <li><a href="">html</a></li>
+  <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+  <li><a href="">bootstrap</a></li>
+  <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+  <li><a href="">Ruby</a></li> 
+  </ul>
+  
+  <!-- SEE PROJECT BUTTON -->
+  <button type="button" class="btn-segund" id="2">See project</button>
   </section>
-  <!-- line4 -->
-   <hr>
-    
-  <footer class="section-footer">
-      <ul class="social-icons">
-        <li><a href=""><img src="./img/Vector.png" alt="github"></a></li>
-        <li><a href=""><img src="./img/Frame.png" alt="Linkedin"></a></li>
-        <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
-        <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
-        <li><a href=""><img src="./imG/M.png" alt="medium"></a></li>
-      </ul>
-    
-      <p class="email-text">emailhere@email.com </p>
-  </footer>
-</body>
-</html>
+
+
+<section class="contact-form"> 
+  <h2 class="contact-us">Contact Us</h2>
+  <p class="paragraph3">If you have an application you are
+    interested in developing, a feature that you <br>
+    need built or a project that needs coding.
+    I'd love to help with it </p>
+
+  <form action="https://formspree.io/forms/meqdknwb/integration" method="post" class="form">
+    <ul>
+      <li>
+        <input type="text" id="jeremy" name="user_name" />
+      </li>
+      <li>
+        <input type="text" id="lastname" name="last_name" />
+      </li>
+
+      <li>
+        <input type="email" id="mail" name="user_email" />
+      </li>
+      <li>
+        <textarea id="msg" name="user_message"></textarea>
+      </li>
+    </ul>
+
+    <li class="button">
+      <button type="submit">Send</button>
+    </li> 
+  </form>
+</form>
+</section>
+<!-- line4 -->
+<hr>
+
+<footer class="section-footer">
+  <ul class="social-icons">
+    <li><a href=""><img src="./img/Vector.png" alt="github"></a></li>
+    <li><a href=""><img src="./img/Frame.png" alt="Linkedin"></a></li>
+    <li><a href=""><img src="./img/Vector5.png" alt="angel"></a></li>
+    <li><a href=""><img src="./img/twitter.png" alt="twitter"></a></li>
+    <li><a href=""><img src="./imG/M.png" alt="medium"></a></li>
```

Comment:
- [ ] The medium link is broken, I think we need to refactor this line of code to help it work.

  ![image](https://user-images.githubusercontent.com/45002654/194877728-65877203-a031-4eea-b629-48a20964fd53.png)
  
  Please refactor this line as below to solve this.
  
```suggestion
    <li><a href=""><img src="./img/M.png" alt="medium"></a></li>
```

In the path definition for the folder name, you have used `imG` with uppercase `G` instead of `img` with lowercase `g`.

Please, also adjust this in the section screenshot below:
![Screenshot 2022-10-10 165236](https://user-images.githubusercontent.com/45002654/194882317-9d74cf54-8457-4503-8d68-9ffb63b8a8b1.png)